src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/IIUnknownStrategy.cs

```diff
@@ -32,7 +32,7 @@ public unsafe interface IIUnknownStrategy
         /// <param name="iid">The IID (Interface ID) to query for.</param>
         /// <param name="ppObj">The resulting interface.</param>
         /// <returns>Returns an HRESULT represents the success of the operation.</returns>
-        /// <seealso cref="Marshal.QueryInterface(nint, in Guid, out nint)"/>
+        /// <seealso cref="Marshal.QueryInterface(nint, ref readonly Guid, out nint)"/>
         public int QueryInterface(void* instancePtr, in Guid iid, out void* ppObj);
```

Comment:
`in` is correct here?

Reply:
That one is. It is a new API and we designed it to be `in`.